From 94e143fc90ff440b45e58c26ecb6dd31ed41bcac Mon Sep 17 00:00:00 2001
From: Danny Trunk <dtrunk90@gmail.com>
Date: Mon, 7 Sep 2020 16:44:24 +0200
Subject: [PATCH 4/4] Changed HID_DEV_MTU_SIZE from 64 to 512

Changed HID_DEV_MTU_SIZE for Amiibo Backups over Bluetooth support

Signed-off-by: Danny Trunk <dtrunk90@gmail.com>
Change-Id: Ie4226b477d040177108729cb25e711fd35bfe0f2
---
 internal_include/bt_target.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/internal_include/bt_target.h b/internal_include/bt_target.h
index 0e153809c..69a177ca1 100644
--- a/internal_include/bt_target.h
+++ b/internal_include/bt_target.h
@@ -1149,7 +1149,7 @@
 #endif
 
 #ifndef HID_DEV_MTU_SIZE
-#define HID_DEV_MTU_SIZE 64
+#define HID_DEV_MTU_SIZE 512
 #endif
 
 #ifndef HID_DEV_FLUSH_TO
-- 
2.17.1

